pilot/pkg/xds/ep_filters_test.go

```diff
@@ -161,15 +164,16 @@ var networkFiltered = []networkFilterCase{
 			{
 				lbEps: []LbEpInfo{
 					// 1 local endpoint on network4
-					{address: "40.0.0.1", weight: 2},
+					{address: "40.0.0.1", weight: 6},
```

Comment:
Why is our  scale much larger now? I thought the lcm logic would help us keep these values lower.

Reply:
I added gateway numbers in the test.

Reply:
saw that once I scrolled a bit more – thanks!